components/badge.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Badges | Docs</title>
+    <link rel="stylesheet" href="../styles/badge.css" />
+    <link rel="stylesheet" href="../styles/avatar.css" />
+    <link rel="stylesheet" href="../styles/button.css" />
+
+    <link rel="stylesheet" href="../docs.css" />
+  </head>
+  <body>
+    <main class="main-wrapper flex flex-col">
+      <!-- navbar starts -->
+      <nav class="nav-wrapper">
+        <div class="container nav-inner-wrapper flex ai-center jc-between">
+          <h1 class="nav-logo">LightUI</h1>
+          <div class="nav-items flex flex-centered">
+            <a href="#!">Docs</a>
+            <a href="#!">Github</a>
```

Comment:
update the links

Reply:
will do it when it will complete thanks.